koku/koku/settings.py

```diff
@@ -382,3 +382,9 @@
 MASU_BASE_URL = 'http://{}:{}/'.format(MASU_SERVICE_HOST, MASU_SERVICE_PORT)
 
 MASU_API_REPORT_DATA = 'api/v1/report_data/'
+
+# AMQP Message Broker
+RABBITMQ_HOST = os.getenv('RABBITMQ_HOST', 'rabbit')
```

Comment:
could you make this default to`localhost`?  It should default to `rabbit` in the docker-compose